.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run precommit
```

Comment:
Could you change this to `npm run lint` and remove `npm run precommit` from the scripts?

Reply:
precommit is the script defined in the package.json which is actuall `npm run lint` to mention what we expect to run before commit the changes . Also precommit is a file which is created in the .husky folder.
I'll try to make the changes you suggested and see if it still works :)